assets/queries/k8s/rbac_wildcard_in_rule/query.rego

```diff
@@ -14,7 +14,7 @@ CxPolicy[result] {
 
 	result := {
 		"documentId": input.document[i].id,
-		"searchKey": sprintf("metadata.name={{%s}}.rules.apiGroups.%s", [metadata.name, document.rules[r].apiGroups[j]]),
```

Comment:
why this search key is less specific?

Reply:
It's impossible to point to the right line, because ```rules``` can have many entries. If the wildcard is present in at least two rules, the search key only points to the first occurrence. Thus, the solution should be to point to the ```rules```.